### PR TITLE
ensure_git_branch action

### PR DESCRIPTION
**Pull request by dbachrach**
Asserts that a lane is run with a specific git branch checked out

### CODE REVIEW

File: lib/fastlane/actions/ensure_git_branch.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+      def self.description
+        "Raises an exception if not on a specific git branch"
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :branch,
+                                       env_name: "FL_ENSURE_GIT_BRANCH_NAME",
+                                       description: "Branch name for EnsureGitBranchAction",
```
Comment (KrauseFx) [inline lib/fastlane/actions/ensure_git_branch.rb]: Please change the description to `The branch that should be checked for` or similar :+1:

**Comment by KrauseFx** [comment]
Thanks for your contribution @dbachrach, this looks great! :+1:

**Comment by dbachrach** [comment]
@KrauseFx addressed the description issue. Thanks.

**Comment by KrauseFx** [comment]
Thank you for your contribution :+1:

**Pull request merged**